RFXtrx/lowlevel.py

```diff
@@ -1120,6 +1122,67 @@ def _set_strings(self):
 
 
 ###############################################################################
+# Bbq class
+###############################################################################
+
+class Bbq(SensorPacket):
+    """
+    Data class for the Temp1 packet type
+    """
+
+    TYPES = {0x01: 'BBQ1 - Maverick ET-732'}
+    """
+    Mapping of numeric subtype values to strings, used in type_string
+    """
+
+    def __str__(self):
+        return ("Bbq [subtype={0}, seqnbr={1}, id={2}, temp1={3}, " +
+                "temp2={4}, battery={5}, rssi={6}]") \
+            .format(self.type_string, self.seqnbr, self.id_string,
+                    self.temp1, self.temp2, self.battery, self.rssi)
+
+    def __init__(self):
+        """Constructor"""
+        super(Bbq, self).__init__()
+        self.id1 = None
+        self.id2 = None
+        self.id3 = None
+        self.id_combined = None
+        self.temp1 = None
+        self.temp2 = None
+        self.battery = None
+
+    def load_receive(self, data):
+        """Load data from a bytearray"""
+        self.data = data
+        self.packetlength = data[0]
+        self.packettype = data[1]
+        self.subtype = data[2]
+        self.seqnbr = data[3]
+        self.id1 = data[4]
+        self.id2 = data[5]
+        self.id3 = data[6]
```

Comment:
Maybe this is wrong?
From domoticz: https://github.com/domoticz/domoticz/blob/master/main/RFXtrx.h#L1603